polytracker/custom_abi/polytracker_abilist.txt

```diff
@@ -3054,6 +3054,7 @@ fun:ptrace=uninstrumented
 fun:ptsname=uninstrumented
 fun:ptsname_r=uninstrumented
 fun:putc=uninstrumented
+fun:putc=custom
```

Comment:
Same as the comment for `dfsan_abilist.txt`

Reply:
same as above